lebedigital/ConcreteOntology/EModuleOntology.ttl

```diff
@@ -126,6 +126,7 @@ con:SecantModulusOfElasticty_ a con:SecantModulusOfElasticty,
     cco:has_decimal_value "$$EModule_Value$$"^^xsd:decimal ;
     cco:has_output con:ExperimentInfo_ ;
     cco:has_process_part con:Computation ;
+    cco:has_text_value "$$URI_Value$$"^^xsd:string ;
```

Comment:
what exactly is stored here? This would then look like "URI_Value" has_text_value "URI_Value", or am I wrong?

Reply:
I renamed the ttl files. The Added URI value describes the testing process of the Ontology (e.g. EModule etc.). As far as I understood this value is needed to later merge the mapped ontologies? The drawio looks something like this:
![Screenshot (52)](https://user-images.githubusercontent.com/113978670/229807057-45f4509d-88e2-47d4-bc8e-7f0e7fcd1add.png)

Reply:
So please talk to @ThiloMuth, @AidaZt  @soudehMasoudian or @alFrie. IMO, we have to store the link (URI) to mix to link those, not the URI of the E-Modulus tests. The type should then also not be string, but the link to a mix. (So take you complete diagram that you have done once, remove all the links that are outside of the E-modulus box (this is what you have done to separate the tasks) and then add again this link that passes from the E-modulus to the mix. This single triple (E-modulus is_made_out_of mix, or whatever that is called).

Reply:
Is this what you mean? The LinkedMix box contains all outside relations of the EModulus and itself is linked to a URI. I still used String as the datatype since I'm not sure what datatype a URI exactly is maybe someone knows this?
![Screenshot (53)](https://user-images.githubusercontent.com/113978670/230066126-2411af7a-ddd0-40fe-a361-6d5ca2e09c2c.png)

Reply:
for the type, can't you use something like: ^^xsd:anyURI?

Reply:
yes that works I seem to have read over this

Reply:
Yes, that sounds good (sorry for the late response).